core/src/main/scala/kafka/server/metadata/BrokerMetadataListener.scala

```diff
@@ -144,6 +144,11 @@ class BrokerMetadataListener(
     thread.doWork()
   }
 
+  // For testing, to verify event queuing
+  private[metadata] def pendingEvents: Int = {
+    queue.size
+  }
+
```

Comment:
Can we change the code `newGauge(BrokerMetadataListener.EventQueueSizeMetricName, () => queue.size)` to use this new method?

Reply:
Ack. Will do